tests/test_data_query.py

```diff
@@ -26,10 +26,10 @@
 import unittest
 import requests
 
+from rcsbsearchapi import rcsb_attributes as attrs
 # from rcsbapi.data import query
 from rcsbapi.data import Schema, Query
-from rcsbsearchapi import rcsb_attributes as attrs
-from rcsbapi.data.schema import PDB_URL
+from rcsbapi.data.constants import ApiSettings
 
 logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s]-%(module)s.%(funcName)s: %(message)s")
 logger = logging.getLogger()
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
And sorry, can you also do this in `tests/test_data_schema.py`? (I can't make a commit suggestion for those lines in that file.)